settings.ini

```diff
@@ -4,7 +4,6 @@ DIAGNOSTICS_VERSION=6f323fe
 CONFIG_VERSION=900aeed
 PKTFWD_VERSION=8aaee4a
 MINER_VERSION=32253cf
-UPNP_VERSION=e6aae2b
```

Comment:
Need to bump config version too before merging

Reply:
Done. But it is the one from my branch. Cool?

Reply:
Got the answer, will wait for merge.

Reply:
Done.